idd/Energy+.idd.in

```diff
@@ -23825,14 +23825,15 @@ AirflowNetwork:MultiZone:WindPressureCoefficientValues,
       \note Enter the WPC Value corresponding to the 36th wind direction.
 
 AirflowNetwork:ZoneControl:PressureController,
-	 \memo Define the Pressures control settings for a zone or list of zones.
-	 \memo If you use a ZoneList in the Zone or ZoneList name field then this definition applies
-	 \memo to all the zones in the ZoneList.
+      \memo This object is used to control a zone to a specified indoor pressure 
+      \memo using the AirflowNetwork model. The specified pressure setpoint is used
+      \memo to control the zone exhaust fan flow rate in a controlled zone or 
+      \memo the relief air flow rate in an air loop.
       \unique-object
  A1 , \field Name
       \required-field
       \reference AirflowNetworkZoneControlPressureControllerNames
- A2 , \field Zone Name
+ A2 , \field Controlled Zone Name
```

Comment:
@mjwitte OK. I changed to "Control Zone Name" in this new feature and will clean up others later.